test/unit/tech/html5.test.js

```diff
@@ -686,7 +688,7 @@ QUnit.test('Html5#reset calls Html5.resetMediaElement when called', function(ass
   Html5.resetMediaElement = oldResetMedia;
 });
 
-QUnit.test('When Android Chrome reports Infinity duration with currentTime 0, return NaN', function(assert) {
+test('When Android Chrome reports Infinity duration with currentTime 0, return NaN', function() {
```

Comment:
why these changes?

Reply:
IE11 was dying here somehow, maybe it was related to the memory issues we fixed and now all this can be reverted